packages/batch-submitter/src/transaction-chain-contract.ts

```diff
@@ -19,6 +20,29 @@ export { encodeAppendSequencerBatch, BatchContext, AppendSequencerBatchParams }
  * where the `appendSequencerBatch(...)` function uses a specialized encoding for improved efficiency.
  */
 export class CanonicalTransactionChainContract extends Contract {
+  public customPopulateTransaction = {
+    appendSequencerBatch: async (
+      batch: AppendSequencerBatchParams,
+      options?: TransactionRequest
```

Comment:
The options aren't really used at all here?

Reply:
Ah good catch, removing the options.